### ISSUE TITLE
Quantized Model is 50% slower in performance. Is this expected?

### ISSUE THREAD

**Issue by mohsinjuni** [tensorflow/tensorflow#19821]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: N/A
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 16.04.4 LTS (Xenial Xerus)
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.7.1
- **Python version**: 3.6.5 :: Anaconda Inc
- **Bazel version (if compiling from source)**: 0.13.1
- **GCC/Compiler version (if compiling from source)**: 5.4.0
- **CUDA/cuDNN version**: N/A
- **GPU model and memory**: GeForce GTX 1080 Ti/PCIe/SSE2
- **Exact command to reproduce**: python tensorflow/examples/label_image/label_image.py

### Describe the problem

Quantized Inception-3 model using transform_graph.py script. Tested on 1000 Imagenet images. Un-quantized model takes 800 seconds whereas quantized model takes 1600 seconds. Is this the expected behavior? 

PS. Let me know please if I should move this to SO?

### Source code / logs

Model files:

>  model_file = \
>     "tensorflow/examples/label_image/data/inception_v3_2016_08_28_frozen.pb"
>   label_file = "tensorflow/examples/label_image/data/imagenet_slim_labels.txt"
> 
> 

```
$ bazel build tensorflow/tools/graph_transforms:transform_graph

$ bazel-bin/tensorflow/tools/graph_transforms/transform_graph --in_graph=./tensorflow/examples/label_image/data/inception_v3_2016_08_28_frozen.pb --out_graph=./tensorflow/examples/label_image/data/eight_bits.pb --inputs=Mul:0 --outputs=final_result:0 --transforms='fold_constants(ignore_errors=true)  fold_batch_norms fold_old_batch_norms quantize_weights'
```

-----------After quantization, I used label_image.py script to test images in a loop as below.

```
  ts = time.time()

  for file in os.listdir('~/test_images'):
      file_name = 'tensorflow/test_images_imagenet/' + file
      print (file_name)
      t = read_tensor_from_image_file(
          file_name,
          input_height=input_height,
          input_width=input_width,
          input_mean=input_mean,
          input_std=input_std)

      with tf.Session(graph=graph) as sess:
          results = sess.run(output_operation.outputs[0], {
          input_operation.outputs[0]: t
      })
      results = np.squeeze(results)

  tf = time.time()
  diff = tf - ts
  print(diff)

```

**Comment by mohsinjuni** [comment]
@tfboyd @petewarden  Could you please guide if this is the best way to do quantization on a pre-trained model. Commands given in [older documentation](https://www.tensorflow.org/versions/r1.3/performance/quantization)  do not work anymore. [New documentation](https://www.tensorflow.org/performance/quantization) seems to expect creating and evaluating graph before running TOCO script. Please help.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @bignamehyp: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @bignamehyp: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by raghuraman-k** [comment]
Quantized inference is now done using tensorflow lite instead of tensorflow core. Please look at the docs at: https://www.tensorflow.org/performance/quantization.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @raghuraman-k: It has been 15 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by suharshs** [comment]
Closing due to inactivity, only ARM optimized kernels exist for quantization at the moment, other kernels can be added but are currently not prioritized.

Please follow the new documentation (https://www.tensorflow.org/performance/quantization) to create a trained quantized model and convert to TFLite with tflite_convert. We will be provided more tools and docs in the upcoming weeks so look out :)

Thanks!
